ghost/i18n/locales/ru/comments.json

```diff
@@ -32,7 +32,7 @@
     "Hidden for members": "Скрыто для участников",
     "Hide": "Скрыть",
     "Hide comment": "Скрыть комментарий",
-    "Jamie Larson": "Павел Бид",
+    "Jamie Larson": "Полное имя",
```

Comment:
This change is not going to be acceptable to the core dev team, where there are strong feelings about having the placeholder be a name, not the words "full name".  So we want something that is recognizably a name.  Ideally gender neutral, but that's not possible in some languages.